frontend/src/components/events/FeatureEvent.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from "react";
+import Moment from "react-moment";
+import defaultLogo from "../../assets/ymim1.png";
+import "./organisms/event_section.css";
+import EventImage from "./atoms/image/image";
+import Text from "./atoms/text/text";
+import Flex from "./atoms/flex/flex";
+
+class FeatureEvent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { isOpen: false };
+  }
+
+  toggle = () => {
+    this.setState({ isOpen: !this.state.isOpen });
+  };
+
+  displayToggle = text => {
+    return (
+      <div>
+        {text}
+        <span
+          onClick={this.toggle}
+          style={{ color: "gray", cursor: "pointer" }}
+          onmouseover=""
```

Comment:
this should be `onMouseOver` not `onmouseover` but I dont think its needed since we are assigning null here